src/main/groovy/com/cloudcard/photoDownloader/CloudCardClient.groovy

```diff
@@ -0,0 +1,117 @@
+package com.cloudcard.photoDownloader
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import kong.unirest.core.HttpResponse
+import kong.unirest.core.Unirest
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Component
+import java.nio.charset.StandardCharsets
+
+import jakarta.annotation.PostConstruct
+
+import static com.cloudcard.photoDownloader.ApplicationPropertiesValidator.throwIfBlank
+
+
+@Component
+//TODO:Not sure if we want the CloucCardClient bean to always be created or not?
+@ConditionalOnProperty(value = "downloader.CloudCardClient", havingValue = "CloudCardClient", matchIfMissing = true)
```

Comment:
Just remove the ConditionalOnProperty annotation. 

It's okay if this bean is in there all the time, for a couple reasons - 1) If we're not using the CloudCardPhotoService, then we're going to be using the SqsPhotoService, or there's something very wrong with the world; 2) There aren't alternatives to the CloudCardClient like there are alternatives to the PhotoService or the StorageService, so having a bean in here that is not getting used will not alter the logic of the code. Whereas in the case of the storage service, having two StorageService beans in the code would result in unpredictable or undesirable wiring - it would be Spring's decision which one would run, not ours.